<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Пользователи</string>
    </property>
    <addaction name="addUserAction"/>
    <addaction name="accessRightsAction"/>
    <addaction name="delUserAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="addUserAction">
   <property name="text">
    <string>Добавить пользователя</string>
   </property>
  </action>
  <action name="delUserAction">
   <property name="text">
    <string>Удалить пользователя</string>
   </property>
   <property name="toolTip">
    <string>Удалить выбранного пользователя</string>
   </property>
  </action>
  <action name="accessRightsAction">
   <property name="text">
    <string>Редактировать права пользователя</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
